MolecularNodes/ui.py

```diff
@@ -452,6 +452,15 @@ def execute(self, context):
         
         return {"FINISHED"}
 
+def menu_residues_selection_custom(layout_function):
+    obj = bpy.context.view_layer.objects.active
+    label = 'Residue selection'
```

Comment:
I think change to 'Res ID Custom' to match `Res ID` and `Res ID Range`.

Reply:
Yeah you are correct that the combination of `MOL_sel_res_id` and `MOL_sel_res_id_range` would be nicer for GN editing. That is why I consider it as a `phenotype`.  I will get it improved soon.

BTW, the input box is absolutely cool. If you want to make a programatic enhancement of MN, I believe that is a good way.

Reply:
Yep I'll definitely see how I can use it in the future!